scan.sh

```diff
@@ -1,35 +1,62 @@
 #!/bin/bash
-set -eux
+set -euo pipefail
 
 MICROSCANNER_TOKEN="${MICROSCANNER_TOKEN:-}"
 DOCKER_IMAGE="${1:-}"
-TEMP_IMAGE_TAG=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr '[:upper:]' '[:lower:]')
-set -o pipefail
+TEMP_IMAGE_TAG=$(tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 32 | head -n 1 | tr '[:upper:]' '[:lower:]' || true)
```

Comment:
oh does this fix the pipefail thing discussed above?

Reply:
Yep, just for uniformity of strict mode headers